### PR TITLE
mgr/dashboard: create subvolume groups

### PR DESCRIPTION
**Pull request by Pegonzal**
Fixes: https://tracker.ceph.com/issues/62363

https://github.com/ceph/ceph/assets/106810838/66818393-0929-45e1-a601-740b80555027

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/shared/api/cephfs-subvolume-group.service.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        mode: mode,
+      },
+      { observe: 'response' }
+    );
+  }
+
+  info(volName: string, groupName: string) {
+    return this.http.get(`${this.baseURL}/${volName}/info`, {
+      params: {
+        subvol_name: groupName
```
Comment (nizamial09) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/api/cephfs-subvolume-group.service.ts:47]:
```suggestion
        group_name: groupName
```

**Review by nizamial09 — COMMENTED** [review]
You might have forgot to commit the changes of the form because its not in here!

**Comment by cloudbehl** [comment]
Hey Pedro, thanks for the PR.

Nit: in some places we are using `Subvolumegroup` I think group is a different word so we can make it `Subvolume group`

**Comment by Pegonzal** [comment]
> Hey Pedro, thanks for the PR. [Screencast from 2023-08-09 15-14-54.webm](https://github.com/ceph/ceph/assets/6695156/9823af34-3aec-4c1f-8059-ee0e6883d195)
> 
> Nit: in some places we are using `Subvolumegroup` I think group is a different word so we can make it `Subvolume group`. Screenshot from 2023-08-09 15-08-46
> 
> Also the notification gives `Created subvolumegroup 'undefined'`.

Thanks Ankush for noticing, I corrected the texts.

**Review by nizamial09 — APPROVED** [review]

**Review by avanthakkar — APPROVED** [review]
LGTM!

**Review by cloudbehl — APPROVED** [review]
LGTM, thanks @Pegonzal

**Pull request merged**